collector/test-data-uploader/src/main/kotlin/com/buildkite/test/collector/android/models/RunEnvironment.kt

```diff
@@ -38,4 +42,11 @@ data class RunEnvironment(
         return ciRunEnvironment
             ?: localRunEnvironment
     }
+
+    companion object {
+        // When bumping version, update VERSION_NAME to match new version
+        // Used for uploading correct library version
+        val VERSION_NAME = "0.2.0-SNAPSHOT"
+        val COLLECTOR_NAME = "test-collector-android"
```

Comment:
Hi @martin-ngyn, I did some work on our other collectors with adding a collector name; the syntax is `{language}-buildkite-test-collector`. Could this be changed to be "android-buildkite-test-collector" to match?

Reply:
Done 🙂